integration_tests/snaps/simple-cmake-replace/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.2)
+project(simple-cmake-replace C)
+configure_file(
+    "${PROJECT_SOURCE_DIR}/config.h.in"
+    "${PROJECT_SOURCE_DIR}/config.h"
+    )
```

Comment:
I think you can get around this by simply doing `add_definitions(-DPART_INSTALL=${PART_INSTALL})`. More info [here](https://cmake.org/cmake/help/v3.0/command/add_definitions.html).